superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -175,8 +176,8 @@ export default class FilterableTable extends PureComponent<
     this.totalTableHeight = props.height;
 
     this.state = {
-      sortDirection: SortDirection.ASC,
       fitted: false,
+      displayedList: [...this.list],
```

Comment:
is this the entirety of the data?

Reply:
Should be all of it. The data gets formatted and set to the `list` variable in the constructor

```javascript
constructor(props: FilterableTableProps) {
    super(props);
    this.list = this.formatTableData(props.data);
```

That's all the data that needs to be sorted from what I can tell. Lemme know if you think I missed something though